integration_tests/webhook/test_webhook.py

```diff
@@ -21,13 +22,7 @@ def setUp(self):
     def tearDown(self):
         pass
 
-    def test_webhook(self):
-        url = os.environ[SLACK_SDK_TEST_INCOMING_WEBHOOK_URL]
-        webhook = WebhookClient(url)
-        response = webhook.send(text="Hello!")
-        self.assertEqual(200, response.status_code)
-        self.assertEqual("ok", response.body)
-
+    def __get_channel_id(self):
```

Comment:
For review: I needed the functionality of pulling channel ID in the two new tests. As this would be repeated code, I have pulled the logic into separate function and use in the `test_webhook`, `test_with_unfurls_off` and `test_with_unfurls_on` methods where needed.